mne/viz/topomap.py

```diff
@@ -805,6 +808,8 @@ def plot_tfr_topomap(tfr, tmin=None, tmax=None, fmin=None, fmax=None,
     fig : matplotlib.figure.Figure
         The figure containing the topography.
     """
+    from ..channels import _get_ch_type
+    ch_type = _get_ch_type(tfr.info, ch_type)
     if format is not None:
         cbar_fmt = format
         warnings.warn("The format parameter is deprecated and will be "
```

Comment:
yeah it is already deprecated, we need to make a sweeping deprecation-cleaning PR after release